src/components/Divider.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Divider as GrafanaDivider, useTheme2 } from '@grafana/ui';
+import { config } from '@grafana/runtime';
+import { isVersionGtOrEq } from '../utils/version';
+
+export function Divider() {
+  const theme = useTheme2();
+  return isVersionGtOrEq(config.buildInfo.version, '10.1.0') ? (
```

Comment:
I think this reads better as just a regular `if`/`else` that returns early if `true`.